src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -68,3 +127,69 @@ export function scrollToFirstError() {
     focusElement(errorEl);
   }
 }
+
+/**
+ * @typedef FormUtility~findTargetsOptions
+ * @type {object}
+ * @property {boolean} getLabel - if true, return label vs actual element
+ */
+/**
+ * Find the wrapping accordion header, or the label of a specific element inside
+ * the accordion
+ * @param {Form~errors} error - single error from `form.formErrors.errors`
+ * @param {FormUtility~findTargetsOptions}
+ * @return {HTMLElement} - either the form element or associated label
+ */
+const findTargets = (error, { getLabel = true } = {}) => {
```

Comment:
Placed in `utilities/ui/index.js`, this `findTargets` doesn't seem to be as generic as the name suggests. What do you think of something more specific such as `findReviewPageErrorTargets`? Alternatively, put this in a module with a more specific path such as `utilities/review/index.js`.

Reply:
👍🏽  Moved the `findTargets` and `focusAndScrollToReviewElement` functions to `utilities/review/index.js`.